docs/visualization/visualize_unit_responses.ipynb

```diff
@@ -83,7 +83,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ax.plot(peak_waveform)
</pre><p>The x-axis seems wrong. I don't think a spike lasts 30 s. Perhaps you need to pass in the x-axis time array ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>